examples/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "license": "MIT",
+  "dependencies": {
+    "@sorare/crypto": "link:/Users/sylvainutard/dev/sorare/crypto/dist",
```

Comment:
This references a local path

Reply:
Shit yes, it's because I didn't publish yet https://github.com/sorare/crypto/pull/1